decidim-core/spec/helpers/decidim/social_share_button_helper_spec.rb

```diff
@@ -61,15 +61,15 @@ module Decidim
       end
 
       context "with X and all optional params" do
```

Comment:
I think this feature in the SocialShareButtonHelper feature we introduced it because it was in the original library that we extracted from. As it isn't (and wasn't) used, we can drop it fully. Let's remove this example

Reply:
Removed